api-tests/.eslintrc.json

```diff
@@ -0,0 +1,229 @@
+{
+  "env": {
+    "browser": true,
+    "node": true
+  },
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "sourceType": "module"
+  },
+  "extends": [
+    "plugin:prettier/recommended",
+    "eslint:recommended",
+    "plugin:vitest/recommended"
+  ],
+  "plugins": [
+    "@typescript-eslint",
+    "import",
+    "formatjs",
+    "no-only-tests",
+    "no-skip-tests",
+    "vitest"
+  ],
+  "rules": {
+    "@typescript-eslint/adjacent-overload-signatures": "error",
```

Comment:
**question**: do you need all those rules? maybe you can use those provided by default eslint plugins?